semcore/tooltip/src/Tooltip.jsx

```diff
@@ -171,9 +173,15 @@ function TooltipPopper(props) {
           use:theme={resolveColor(theme)}
           use:aria-live={undefined}
           ref={popperRef}
+          __excludeProps={['arrowBgColor', 'arrowShadowColor']}
```

Comment:
It should not be applied anyway due to propsForElement filter

Reply:
removed